bp5/bench.cc

```diff
@@ -199,7 +237,7 @@ void test(const unsigned int s,
       laplace_operator.vmult_add(tmp, output);
       deallog << "True residual norm: " << tmp.l2_norm() << std::endl;
     }
-  AssertThrow(solver_control.last_step() == iterations_basic,
+  AssertThrow(true || solver_control.last_step() == iterations_basic,
```

Comment:
This assertion is too unstable - would you mind removing it or relax it by requiring += 2 iterations around that value? (And for the other appearances as well)? Thanks!

Reply:
I'll do!